packages/babel-parser/typings/babel-parser.d.ts

```diff
@@ -238,4 +238,4 @@ type ParseResult<Result> = Result & {
   errors: ParseError[];
 };
 
-export { DecoratorsPluginOptions, FlowPluginOptions, ParseError, ParseResult, ParserOptions, ParserPlugin, ParserPluginWithOptions, PipelineOperatorPluginOptions, RecordAndTuplePluginOptions, TypeScriptPluginOptions, parse, parseExpression, tokTypes };
+export { type DecoratorsPluginOptions, type FlowPluginOptions, type ParseError, type ParseResult, type ParserOptions, type ParserPlugin, type ParserPluginWithOptions, type PipelineOperatorPluginOptions, type RecordAndTuplePluginOptions, type TypeScriptPluginOptions, parse, parseExpression, tokTypes };
```

Comment:
Is this a breaking change for us?🤔

Reply:
Technically yes -- instead of having two separate files, can we just to `.replace(/([{,]\s*)type\s*/g, "$1")` in this file?

Reply:
I realized it also affects `@babel/types`, so now I use both versions.